Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.SerialWombat/Samples/Sw18AB_Ultrasonic_Sample/Sw18AB_Ultrasonic_Sample.csproj

```diff
@@ -22,6 +22,7 @@
     <ProjectReference Include="..\..\..\..\..\..\Meadow.Core\Source\Meadow.F7\Meadow.F7.csproj" />
     <ProjectReference Include="..\..\..\..\..\..\Meadow.Logging\lib\Meadow.Logging.csproj" />
     <ProjectReference Include="..\..\..\..\..\..\Meadow.Units\Source\Meadow.Units\Meadow.Units.csproj" />
+    <ProjectReference Include="..\..\..\Sensors.Distance.Hcsr04\Driver\Sensors.Distance.Hcsr04.csproj" />
```

Comment:
is this the right project reference? Shouldnt it be SerialWombat or similar?

Reply:
Yeah it actually is correct - there's a wombat specific implementation for that exact driver. It's actually implemented in the firmware - odd / interesting right?